src/dot.github.workflows.main.yml.gen.rb

```diff
@@ -30,16 +30,20 @@
           sudo docker run --privileged --name qr -e CI=true -t quine-relay
       - name: push spoiler
         run: |
-          mkdir spoiler
+          git clone https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git spoiler --branch spoiler
+          git -C spoiler rm --quiet -r '*'
 #{ cp_cmds }
           cd spoiler
-          git init --quiet
-          git config user.name 'Yusuke Endoh'
-          git config user.email 'mame@ruby-lang.org'
           git add .
-          git commit -m spoiler --quiet
-          git push --force --quiet https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/mame/quine-relay.git master:spoiler
-          echo The intermediate sources are available: https://github.com/mame/quine-relay/tree/spoiler
+          GIT_AUTHOR_NAME="$(git show -s --format=%an "$GITHUB_SHA")" \
+          GIT_AUTHOR_EMAIL="$(git show -s --format=%ae "$GITHUB_SHA")" \
+          GIT_AUTHOR_DATE="$(git show -s --format=%ad "$GITHUB_SHA")" \
+          GIT_COMMITTER_NAME='GitHub Actions' \
+          GIT_COMMITTER_EMAIL='actions@github.com' \
```

Comment:
Is it common GHA practice to use this email address when committing automatically in GHA? I am a little uncomfortable with making a commit with someone else's email address.

Reply:
They seem to encourage using `GitHub Actions <actions@github.com>` in their [documentation](https://github.com/marketplace/actions/add-commit#different-authorcommitter-configurations) for scenarios similar to this.

Reply:
I see, thanks. Sounds good.